esxiSsh.py

```diff
@@ -0,0 +1,288 @@
+# Created by Aaron Soto (@_surefire_)
+# in collaboration with Brendan Watters (@tychos_moose)
+# released under the BSD license
+# v0.1 (2017-June-25)
+
+# TODO: Test on ESXi 6.0 and 5.5 (only tested with ESXi 6.5)
+# TODO: Confirm that snapshots work
+# TODO: Determine whether VMX hardware changes between snapshots are copied over
+# TODO: Allow cloining from a previous source snapshot to a new destination machine
+
+import esxiVm
+import pexpect
+import sys
+from pyVmomi import vim
+
+class esxiSsh(esxiVm.esxiServer):
+    def copyOvfTool(self):
+        # TODO: Detect if this version of ovftool is already present on remote server, then return true
+
+        ovfToolsPath = './ovftool-4.2.0-4586971.tgz'
+        ovfToolsURL = 'https://my.vmware.com/group/vmware/details?downloadGroup=OVFTOOL420&productId=491'
```

Comment:
Once logged in the link here provides `bundle`, `dmg` and `msi` format packages however no `tgz` is provided, what is the expectation here?  Are there additional steps needed to extract the `tgz` that is needed here?